.gitignore

```diff
@@ -39,3 +39,6 @@ provision-xcode.csx
 mono_crash.*.json
 *.binlog
 .vscode
+tests/BundledResources/BundledResources.csproj
+tests/EmbeddedResources/EmbeddedResources.csproj
+tests/fsharplibrary/fsharplibrary.fsproj
```

Comment:
This change is not related to this PR, so it should be removed.

If needed, it should go in a separate PR, explaining the reason for the change (I'm also a bit puzzled about this change, because these files are already checked into git, so they shouldn't be ignored, nor should this actually change any behavior),